Makefile

```diff
@@ -81,10 +86,19 @@ localeextract:
 	python manage.py extract
 	python manage.py merge
 
+locale-populate-react:
+	@ mkdir -p locale/compendia
```

Comment:
If you add an `rmdir -rf` at the end of this command, then you won't have to add that in the Dockerfile below...  It doesn't look like the compendium files get used after they are created, so it is unclear to me that it is worth keeping them around. I suppose you could also use temp files to hold the compendia... But creating a special-purpose directory as you do here seems simpler to me.